Samples/MyTestiOSApp/MyTestiOSApp/ADTestAppProfileViewController.m

```diff
@@ -0,0 +1,134 @@
+//
+//  ADTestAppProfileViewController.m
+//  MyTestiOSApp
+//
+//  Created by Ryan Pangrle on 4/29/16.
+//  Copyright © 2016 MS. All rights reserved.
+//
+
+#import "ADTestAppProfileViewController.h"
+#import "ADTestAppSettings.h"
+
+static NSDictionary* s_profiles = nil;
+static NSArray* s_profileTitles = nil;
+
+@interface ADTestAppProfileViewController ()
+
+@end
+
+@implementation ADTestAppProfileViewController
+{
+    UITableView* _profileTable;
+}
+
++ (void)initialize
+{
+    s_profiles =
+    @{ @"Test App"    : @{ @"authority" : @"https://login.microsoftonline.com/common",
+                           @"resource" : @"https://graph.windows.net",
+                           // NOTE: The settings below should come from your registered application on
+                           //       the azure management portal.
+                           @"clientId" : @"b92e0ba5-f86e-4411-8e18-6b5f928d968a",
+                           @"redirectUri" : @"x-msauth-adaltestapp-210://com.microsoft.adal.2.1.0.TestApp",
+                           },
+       @"Office"      : @{ @"authority" : @"https://login.microsoftonline.com/common",
+                           @"resource" : @"https://api.office.com/discovery",
+                           @"clientId" : @"d3590ed6-52b3-4102-aeff-aad2292ab01c",
+                           @"redirectUri" : @"urn:ietf:wg:oauth:2.0:oob",
```

Comment:
And no, I'm not particularly interested in testing out every possible permutation of input on the broker from the test app. Testing that we can talk to broker and get a response back is sufficient from a local ADAL perspective.